app/src/main/java/io/github/droidkaigi/confsched2018/presentation/detail/SessionDetailFragment.kt

```diff
@@ -48,13 +48,23 @@ class SessionDetailFragment : Fragment(), Injectable {
             when (result) {
                 is Result.Success -> {
                     val sessions = result.data
+                    val position = sessions.indexOfFirst { it.id == sessionId }
                     bindSession(sessions.first { it.id == sessionId })
```

Comment:
Could you please modify here to `sessions[position]` too?

Reply:
I didn't realize that, thank you 👍